.github/workflows/repo2cpe-download.yaml

```diff
@@ -0,0 +1,49 @@
+name: Download Repo2CPE and Name2CPE
+on:
+  push:
+    branches:
+    - "yli3/**"
+  schedule:
+  - cron: "0 */3 * * *"
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Authenticate with Google Cloud
+      uses: google-github-actions/auth@v1
+      with:
+        credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+
+    - name: Create Repo2CPE directory
+      run: mkdir -p Repo2CPE
```

Comment:
Why do we need a separate step to `Create .* directory` m instead of making this part of the `wget` script?